### PR TITLE
Yr/ convert bc108 and bc109 to the new validator

### PR DESCRIPTION
**Pull request by RosenbergYehuda**
## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9827
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-9804

## Description
- Converted BC108 and BC 109 to the new validate. (merged together to be  BC108)
- Added this validation to apply on playbooks and pacs as well.
- Added functionality to this validation to catch renamed (moved to a different pack) content items as well.

### CODE REVIEW

**Review by YuvHayun — CHANGES_REQUESTED** [review]
Looks good.
I added a few comments.
Also, let's remove the use of conditions inside the tests.
There shouldn't be any logic & conditions inside the tests. If needed, add more test functions, but right now it's really hard to read, debug, and undersstand.

File: demisto_sdk/commands/validate/validators/BC_validators/BC108_was_marketplace_modified.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+]
+ALL_MARKETPLACES = list(MarketplaceVersions)
+class WasMarketplaceModifiedValidator(BaseValidator[ContentTypes]):
+    error_code = "BC108"
+    description = "Ensuring that the 'marketplaces' property hasn't been removed or added in a manner that effectively removes all others."
+    error_message = "You are not permitted to add (in a manner that directly removes) or delete the 'marketplaces' field from current content. Either revert the action or request a force merge."
+    fix_message = ""
+    related_field = ""
+    is_auto_fixable = False
+    expected_git_statuses = [GitStatuses.MODIFIED]
```
[the remark this answers is not in the document]
Comment (RosenbergYehuda) [reply demisto_sdk/commands/validate/validators/BC_validators/BC108_was_marketplace_modified.py:40]: Why? If you only renamed the file, how is that relevent to this validation?

File: demisto_sdk/commands/validate/tests/test_tools.py [hunk 2]
```diff
@@ -49,6 +50,25 @@ def create_integration_object(
     return BaseContent.from_path(Path(integration.path))  # type:ignore
 
 
+def create_pack_object(
```
[the remark this answers is not in the document]
Comment (RosenbergYehuda) [reply demisto_sdk/commands/validate/tests/test_tools.py:53]: I missed that one....

File: demisto_sdk/commands/validate/validators/BC_validators/BC108_was_marketplace_modified.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (RosenbergYehuda) [reply demisto_sdk/commands/validate/validators/BC_validators/BC108_was_marketplace_modified.py]: Done

File: demisto_sdk/commands/validate/tests/test_tools.py [hunk 3]
```diff
@@ -10,6 +10,7 @@
 from demisto_sdk.commands.common.tools import set_value
 from demisto_sdk.commands.content_graph.objects.base_content import BaseContent
 from demisto_sdk.commands.content_graph.objects.integration import Integration
+from demisto_sdk.commands.content_graph.objects.pack import Pack
```
Comment (YuvHayun) [inline demisto_sdk/commands/validate/tests/test_tools.py:13]: ?

**Review by YuvHayun — APPROVED** [review]

**Pull request merged**